recipes/pulseaudio/meson/conanfile.py

```diff
@@ -0,0 +1,173 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import cross_building
+from conan.tools.env import VirtualBuildEnv, VirtualRunEnv
+from conan.tools.files import copy, get, rm, rmdir
+from conan.tools.gnu import PkgConfigDeps
+from conan.tools.meson import Meson, MesonToolchain
+from conan.tools.layout import basic_layout
+import os
+
+required_conan_version = ">=1.53.0"
+
+
+class PulseAudioConan(ConanFile):
+    name = "pulseaudio"
+    description = "PulseAudio is a sound system for POSIX OSes, meaning that it is a proxy for sound applications."
+    topics = ("sound",)
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://pulseaudio.org/"
+    license = "LGPL-2.1"
+
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_alsa": [True, False],
+        "with_glib": [True, False],
+        "with_fftw": [True, False],
+        "with_x11": [True, False],
+        "with_openssl": [True, False],
+        "with_dbus": [True, False],
+        "with_gdbm": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_alsa": True,
+        "with_glib": True,
+        "with_fftw": False,
+        "with_x11": True,
+        "with_openssl": True,
+        "with_dbus": False,
+        "with_gdbm": False,
+    }
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+        self.settings.rm_safe("compiler.libcxx")
+        self.settings.rm_safe("compiler.cppstd")
+        if not self.options.with_dbus:
+            del self.options.with_fftw
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires("libiconv/1.17")
+        self.requires("libsndfile/1.2.0")
+        self.requires("libcap/2.66")
+        self.requires("libtool/2.4.7")
+        if self.options.with_alsa:
+            self.requires("libalsa/1.2.7.2")
+        if self.options.with_glib:
+            self.requires("glib/2.76.1")
+        if self.options.get_safe("with_fftw"):
+            self.requires("fftw/3.3.10")
+        if self.options.with_x11:
+            self.requires("xorg/system")
+        if self.options.with_openssl:
+            self.requires("openssl/3.1.0")
```

Comment:
```suggestion
            self.requires("openssl/[>=1.1 <4]")
```

Recent change of CCI policies. See https://github.com/conan-io/conan-center-index/pull/17192